modules/jsf/src/main/java/org/jboss/weld/module/jsf/FacesUrlTransformer.java

```diff
@@ -31,6 +33,7 @@
     private static final String QUERY_STRING_DELIMITER = "?";
     private static final String PARAMETER_PAIR_DELIMITER = "&";
     private static final String PARAMETER_ASSIGNMENT_OPERATOR = "=";
+    private static final Pattern cidParamPattern = Pattern.compile("(\\?|&)cid=");
```

Comment:
It's good to name constants like `CID_PARAM_PATTERN`...

Reply:
Ah, good point, I extracted it from method and forgot to change it.